### PR TITLE
Visitor can see and navigate via footer

### PR DESCRIPTION
**Pull request by leiter007**
# [PT Story](https://www.pivotaltracker.com/story/show/166572746)

## Changes proposed in this pull request:
* Integration test for seeing headline in footer, and showing menu sidebar
* Adds component MenuSidebar, showing when clicked on hamburger menu icon in Footer
* Adds global site variable for color "orange" to be able to adjust CSS fontcolor across the App
* Removes MainContent component and puts Map in App.js to access props across the app  

## What I have learned working on this feature:
* Creating sidebar in Semantic UI
* Responsiveness of different screen sizes
* Using Semantic UI site variables for color

## Screenshot:
Screenshot 2019-06-14 at 15 46 38
Screenshot 2019-06-14 at 15 46 55

### CODE REVIEW

**Comment by zanenkn** [comment]
it is a beautiful thing you guys have made!!!!

Although. I pulled it down and tested in different formats. noticed those borders at the footer, imo unnecessary and in certain formats look weird, for example:

**Comment by zanenkn** [comment]
image

**Comment by zanenkn** [comment]
I suggest we take those away (sorry Felix, i know you love your dividers)

**Comment by leiter007** [comment]
> I suggest we take those away (sorry Felix, i know you love your dividers)

I know, a bit strange - trying to find where this is set. Because it is actually NOT any of my lovely dividers.. :)

**Comment by zanenkn** [comment]
i figured, dividers are ugly but at least you place them evenly, this didnt look like your doing..

File: src/Components/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import { Grid, Icon, Header, Container } from 'semantic-ui-react';
+
```
Comment (raptorf1) [inline src/Components/Footer.jsx:3]: `Container` is not used in this component

**Review by raptorf1 — CHANGES_REQUESTED** [review]

File: src/Components/Footer.jsx (hunk 1 above)
Comment (raptorf1) [reply src/Components/Footer.jsx:3]: removed

**Pull request merged**